libraries/AP_Mount/AP_Mount_Siyi.cpp

```diff
@@ -74,6 +74,12 @@ void AP_Mount_Siyi::update()
 #endif
     }
 
+    if (!_codec_info.main_stream.received || !_codec_info.recording_stream.received) {
+    // Request codec specs at startup
```

Comment:
Txs for this.  nitpick: little indenting issue here..

Reply:
> Txs for this. nitpick: little indenting issue here..

Thanks, fixed it.